src/components/GraphSection/Constants/graphConstants.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphType } from "../Types/GraphType";
+import { TeamData } from "../../../services/ProjectTeams";
```

Comment:
Configure typescript and define a root folder so it's easier to define absolute paths
```suggestion
import { TeamData } from "services/ProjectTeams";
```